create_config_file.py

```diff
@@ -0,0 +1,69 @@
+import json
+import sys
+
+both = {
+    'vars': {
+        'hash': None,
+        'driver_memory': None,
+        'spark': None,
+        'image': None
```

Comment:
what's the significant of putting these in? is it intended as documentation of the available vars?